lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,36 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    class ListCommand < HammerCLIForeman::ListCommand
+
+      output do
+        field :id, _("Id")
+        field :created_at, _("At")
+        field :remote_address, _("IP")
+        field :user_name, _("User name")
+        field :action, _("Action")
+        field :auditable_type, _("Audit type")
+        field :auditable_name, _("Audit name")
+      end
+
+      build_options
+    end
+
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
```

Comment:
Currently there's not much added value in the info command as it prints the same as list. I was thinking about providing some more details about the changed fields and values there. Similar to what UI does:
```
Default changed from --- true ... to --- false ...
Value changed from N/A to --- true ...
```
What do you think?

It will probably require some extending/mapping of the data that come form api. Hammer commands have method `extend_data(record)` for that. Let me know in case you need some help with that.